app/main.py

```diff
@@ -1,8 +1,54 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        if 1 <= comfort_class <= 7:
+            self.comfort_class = comfort_class
+
+        if 1 <= clean_mark <= 10:
+            self.clean_mark = clean_mark
+
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        if 1.0 <= distance_from_city_center <= 10.0:
+            self.distance_from_city_center = distance_from_city_center
+
+        self.clean_power = clean_power
+
+        if 1.0 <= average_rating <= 5.0:
+            self.average_rating = average_rating
+
```

Comment:
what if value is incorrect? it means class will not have these attributes at all

Reply:
I thought we have to control for producing only objects with good parameters.